openlibrary/plugins/openlibrary/js/SearchUtils.js

```diff
@@ -0,0 +1,136 @@
+import { removeURLParameter } from './Browser';
+import $ from 'jquery';
+
+/**
+ * Adds hidden input elements/modifes the action of the form to match the given search mode
+ * (Don't really understand what's happening, so just not gonna touch it too much)
+ * @param {JQuery} form the form we'll modify
+ * @param {String} searchMode
+ */
+export function addModeInputsToForm($form, searchMode) {
+    $('input[value=\'Protected DAISY\']').remove();
+    $('input[name=\'has_fulltext\']').remove();
```

Comment:
These are global jQuery selectors. I think these should be moved out of this function as it will make this function purely about the form and adding inputs to it as the name subject.